src/redux/slices/userSlice.ts

```diff
@@ -15,8 +17,10 @@ export const userSlice = createSlice({
     userLogin: (state, action: PayloadAction<UserState>) => {
       const profile = action.payload.profile;
       const graph = action.payload.graph;
+      const wallet = action.payload.wallet;
       state.profile = profile;
       state.graph = graph;
+      state.wallet = wallet;
```

Comment:
Not really related, but why not:
```suggestion
      state.profile = action.payload.profile;
      state.graph = action.payload.graph;
      state.wallet = action.payload.wallet;
```